widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterController.kt

```diff
@@ -76,6 +77,7 @@ class MessageCenterController(
     @VisibleForTesting
     fun handleSendMessageResult(gliaException: GliaException?) {
         if (gliaException == null) {
+            setSecureEngagementUseCase(true)
```

Comment:
Sorry, I don't understand the logic here. Each time any message-sending result is handled and gliaException is null, we set `SecureEngagementUseCase` to `true`?

Reply:
Why each time? Only when we success sends the message from the Welcome to Message Center screen.

Reply:
Ah, I forgot that `MessageCenterController` is related only to secure messaging. I mistakenly thought that `MessageCenterController` refers to all messages and that's why misunderstood the logic.